entries/t/txbit.io.json

```
@@ -0,0 +1,12 @@
+{
+  "Txbit": {
+    "domain": "txbit.io",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://support.txbit.io/support/solutions/articles/44000447137",
+    "keywords": [
+      "cryptocurrency"
```

```suggestion
      "cryptocurrencies"
```
The keywords should follow the names of the categories, so this should be `cryptocurrencies`.
https://github.com/2factorauth/twofactorauth/blob/6afe487872381a1d74cad461d849e2ee0602c2ff/_data/categories.json#L33